CryptKeeperZK/crypt-keeper-extension
# feat: demo host connection

## Explanation

This PR is a part of #445 adds support for identity host connection with demo host.

Details are below:
- [x] Add proof output section
- [x] Support identity creation with host
- [x] Move manifest files to project root
- [x] Add license and privacy policy to dist
- [x] Rename selected identity to connected
- [x] Support search params for popup redirection
- [x] Update demo types

## More Information

Related to #445 

## Screenshots/Screencaps

![image](https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/14254374/e35fb2e6-cd04-4197-9e58-9d6368abc4b5)

## Manual Testing Steps

1. Check identities created from demo have host in metadata. 
2. Check create identity page is open properly from extension

## Pre-Merge Checklist


> PR template source from [github.com/MetaMask](https://github.com/MetaMask)
